.travis.yml

```diff
@@ -5,6 +5,15 @@ scala:
   - 2.11.8
 jdk:
   - oraclejdk8
+cache:
+  branch: md5deep
```

Comment:
Curious-- what does this "cache: branch: md5deep" line do? Have a link to example/docs?